src/Mod/Measure/Gui/QuickMeasure.h

```diff
@@ -45,15 +45,19 @@ class QuickMeasure : public QObject, Gui::SelectionObserver
 
 private:
     void onSelectionChanged(const Gui::SelectionChanges& msg) override;
-    void tryMeasureSelection(const Gui::SelectionChanges& msg);
+    void tryMeasureSelection();
 
     bool canMeasureSelection(const Gui::SelectionChanges& msg) const;
     void addSelectionToMeasurement();
     void printResult();
     void print(const QString& message);
 
+    void processSelection();
+
     Measure::Measurement* measurement;
 
+    QTimer* selectionTimer;
```

Comment:
A forward declaration of QTimer is needed.

Reply:
added